helper/resource/plugin.go

```diff
@@ -103,7 +125,8 @@ func runProviderCommand(f func() error, wd *tftest.WorkingDir, opts *plugin.Serv
 
 	// wait for the server to actually shut down; it may take a moment for
 	// it to clean up, or whatever.
-	<-closeCh
+	// TODO: add a timeout here?
```

Comment:
Select and race a timeout might be good so acceptance tests don't hang. Make sure to log or error though because this should not happen

Reply:
This could happen due to a provider implementation error, however.

Reply:
(As I noted in a comment on the code, won't the individual test timeout be sufficient for this? Or would we like our own, more aggressive timeout?)

Reply:
That's what I wasn't sure about, go test already times out, but its an aggregate timeout for all tests, not for an individual I believe. It may be useful to add an individual timeout on the `TestCase` or something? Unsure and people have some long tests, so didn't want to cause a need for additional modification to get it to work for those cases that exist.